src/main/groovy/org/mockito/release/internal/gradle/ContinuousDeliveryPlugin.java

```diff
@@ -89,11 +91,19 @@ public void execute(Task t) {
         //TODO can we make git push and bintray upload tasks to be last (expensive, hard to reverse tasks should go last)
 
         project.allprojects(new Action<Project>() {
-            public void execute(final Project project) {
-                project.getPlugins().withType(BintrayPlugin.class, new Action<BintrayPlugin>() {
+            public void execute(final Project p) {
```

Comment:
I'll see what I can do ;) I like single letter variables for tiny methods. Thanks for feedback, though!